migrations/2021-04-24-174047_add_show_read_post_setting/up.sql

```diff
@@ -0,0 +1 @@
+alter table local_user add column show_read_posts boolean default true not null;
```

Comment:
I could've done "hide read", but all the other ones are named `show`, so this seems more consistent.